src/components/AudioLevelIndicator/AudioLevelIndicator.tsx

```diff
@@ -52,29 +52,24 @@ function AudioLevelIndicator({
       // and switches back to the app.
       window.addEventListener('focus', reinitializeAnalyser);
 
-      function render() {
-        animationFrameID = window.requestAnimationFrame(() => {
-          const array = new Uint8Array(analyser.frequencyBinCount);
-          analyser.getByteFrequencyData(array);
-          let values = 0;
+      const timer = interval(() => {
```

Comment:
What's the difference between this and setInterval?

Reply:
It's like `setInterval`, but it uses `requestAnimationFrame` under the hood. This makes it good for animations, and it also guarantees that it won't run in the background.

Reply:
What is the 2nd parameter for? Is that 50ms? Basically, it will use requestAnimationFrame but it won't run more than once within a 50ms time period?

Reply:
Correct. It's like `window.setTimeout(func, 50)`.

https://github.com/d3/d3-timer#interval

Reply:
Ok cool. I think this looks good 👍